modules/logging-rn-5.5.7.adoc

```diff
@@ -0,0 +1,22 @@
+//module included in logging-5-5-release-notes.adoc
+:_content-type: REFERENCE
+[id="logging-release-notes-5-5-7_{context}"]
+= Logging 5.5.7
+This release includes link:https://access.redhat.com/errata/RHSA-2023:0633[OpenShift Logging Bug Fix Release 5.5.7].
+
+[id="logging-5-5-7-bug-fixes"]
+== Bug fixes
+* Before this update, the LokiStack Gateway Labels Enforcer generated parsing errors for valid LogQL queries when using combined label filters with boolean expressions. With this update, the LokiStack LogQL implementation supports label filters with boolean expression and resolves the issue. (link:https://issues.redhat.com/browse/LOG-3534[LOG-3534])
+
+* Before this update, the `ClusterLogForwarder` custom resource (CR) did not pass TLS credentials for syslog output to Fluentd, resulting in errors during forwarding. With this update, credentials pass correctly to Fluentd, resolving the issue. (link:https://issues.redhat.com/browse/LOG-3533[LOG-3533])
```

Comment:
"for syslog output to Fluent" reads like syslog output goes fo Fluentd, while actually it's credentials. But I am not sure how to fix this.